dashboard/config/callouts.tsv

```diff
@@ -15,8 +15,7 @@ Show Code 20	20-hour	2_13	Maze	show_code_explanation	#show-code-header	{"positio
 Show Code HoC	Hour of Code	2_13	Maze	show_code_explanation	#show-code-header	{"position": {"my": "center right", "at": "center left"}}
 Instructions 20	20-hour	2_14	Maze	instructions_repeat_explanation	#prompt	{"position": {"my": "bottom left", "at": "top center"}}
 Instructions HoC	Hour of Code	2_14	Maze	instructions_repeat_explanation	#prompt	{"position": {"my": "bottom left", "at": "top center"}}
-Category 20	20-hour	2_1	Artist3	category_explanation	.blocklyTreeRoot	{"position": {"my": "center left", "at": "center right"}}
```

Comment:
This callout works with CDO Blockly only, and was duplicated below when we were working on migrating Artist.